ci/travis/build-docker.sh

```diff
@@ -16,20 +16,20 @@
 
 set -eu
 
-readonly BINDIR="$(dirname $0)"
-source "${BINDIR}/../colors.sh"
-
 # This script is supposed to run inside a Docker container, see
 # ci/travis/build-linux.sh for the expected setup.  The /v directory is a volume
 # pointing to a (clean-ish) checkout of google-cloud-cpp:
-(cd /v ; ./ci/check-style.sh)
+if [[ -z "${PROJECT_ROOT+x}" ]]; then
+  readonly PROJECT_ROOT="/v"
+fi
+source "${PROJECT_ROOT}/ci/travis/linux-config.sh"
+source "${PROJECT_ROOT}/ci/colors.sh"
 
 # Run the configure / compile / test cycle inside a docker image.
 # This script is designed to work in the context created by the
 # ci/Dockerfile.* build scripts.
-readonly IMAGE="cached-${DISTRO}-${DISTRO_VERSION}"
-readonly BUILD_DIR="build-output/${IMAGE}"
-readonly STAGING_DIR="build-output/${IMAGE}-staging"
+
+(cd /v ; ./ci/check-style.sh)
```

Comment:
cd "${PROJECT_ROOT" ?

Reply:
Done.

I search for others, only found them in sed expressions where using the variable would be more hassle than it is worth.